src/mappers/jsonMapper.ts

```diff
@@ -0,0 +1,17 @@
+import { ValueMapper } from '../abstracts/ValueMapper';
+
+const parseJSON = (value: string): IJsonResult | null => {
+    try {
+        return JSON.parse(value)
+    } catch (e) {
+        return null
+    }
+}
+
+interface IJsonResult {
+    [key: string]: string | boolean | number
+}
+
+type JsonMapper = ValueMapper<IJsonResult | null>
+
+export const jsonMapper: JsonMapper = parseJSON
```

Comment:
I consider about one of two way to implement jsonMapper. Please choose this one which you prefer:

1. The simplest, enough for in `xlsx-import` usages
```ts
export const jsonMapper: ValueMapper<any> = (value: string): any => {
    try {
        return JSON.parse(value)
    } catch (e) {
        return null
    }
}
```

2. More complex, and better typed way:

```ts
type JsonMapper= <TJsonResult>(value: string) => TJsonResult|null;

export const jsonMapper:JsonMapper = <TJsonResult extends any>(value: string): TJsonResult|null => {
    try {
        return JSON.parse(value)
    } catch (e) {
        return null
    }
}```

Reply:
Hi @Siemienik,

thank you for your great support! I understood it now 🙂